index.js

```diff
@@ -1011,10 +1022,13 @@
   //.
   //. > S.Just(S.Right(42)).sequence(S.Either.of)
   //. Right(Just(42))
+  //.
+  //. > S.Just(S.Left('Cannot divide by zero')).sequence(S.Either.of)
+  //. Left('Cannot divide by zero')
   //. ```
   Maybe.prototype.sequence =
   method('Maybe#sequence',
-         {a: [Apply], b: [Apply]},
+         {a: [Applicative], b: [Applicative]},
```

Comment:
> It's arguably correct either way, given that the function to make the `Apply` and `Applicative` is explicitly provided by the caller.

The caller could provide her own `of`, certainly. Presumably, though, the caller will simply provide `Future.of` or whatever.

If it's correct either way, let's go with the option which matches our type signature. :)